static/bidder-info/mediafuse.yaml

```diff
@@ -1,17 +1,15 @@
 maintainer:
   email: "support@mediafuse.com"
-gvlVendorID: 411
```

Comment:
This is an invalid GVL vendor id. I couldn't find Mediafuse listed. Are you listed under a different name?

Reply:
Sorry, where to get this GVL ID from. Can you mention some details

Reply:
Sorry, where to get this GVL ID from. Can you mention some details

Reply:
The GVL ID is your company's id in the IAB Europe TCF 2.0 Vendor List: https://iabeurope.eu/vendor-list-tcf-v2-0

You should get someone from your company's legal / privacy team to help you here.

Reply:
I shall check with my Management and get back at the earliest

Reply:
@SyntaxNode, I spoke to the relevant person in my company and it follows

We're not part of the [list](https://iabeurope.eu/vendor-list-tcf-v2-0). Most of our pubs are also US-based, although we will likely have more traffic from Europe in the future. Does it require to be on the list to make further progress? 
Is this list applies to US vendors too?
What would be the impact to our S2S setup if we were not on the list?

Reply:
@SyntaxNode, I spoke to the relevant person in my company and it follows

We're not part of the [list](https://iabeurope.eu/vendor-list-tcf-v2-0). Most of our pubs are also US-based, although we will likely have more traffic from Europe in the future. Does it require to be on the list to make further progress?
Is this list applies to US vendors too?
What would be the impact on our S2S setup if we were not on the list?

Reply:
What happened to the GVL ID. Wrapper code was approved without a GVL ID or there was an alternate solution to bypass GVL ID?

Reply:
> Does it require to be on the list to make further progress?

No. We do not require adapters to have a registered GVL ID. I just want to make sure you considered what the appropriate value should be from your compliance / legal / privacy team.

> Is this list applies to US vendors too?

It applies to any auction in which the publisher has elected that GDPR should be enforced.

> What would be the impact on our S2S setup if we were not on the list?

If a publisher specifies a GDPR consent string, there would be no way for that publisher to reference your adapter. Please refer to https://iabeurope.eu/tcf-2-0/ for further details.